### PR TITLE
Update and review changes

### PR DESCRIPTION
**Pull request by mmitoraj**
- Language review
- Content update

### CODE REVIEW

**Comment by mmitoraj** [comment]
WIP

File: rafter/step1.md [hunk 1]
```diff
@@ -1,22 +1,7 @@
-WHAT ARE THE PRE_STEPS A USER CAN SEE IN THE TERMINAL? IS IT HELM INSTALLATION / TILLER INSTALLATION / MINIKUBE SET-UP? 
-Make sure the environment is set up. Check if the API Server is running. 
-It make take at least 1 minute, Minikube needs time, be patient.
-`kubectl get pod -n kube-system | grep kube-apiserver-minikube`{{execute}} - DOES THE COMMAND CHECK IF MINKUBE IS SET UP CORRECTLY?
+The environment is set up and ready to start palying with Rafter.
```
Comment (derberg) [inline rafter/step1.md]: rather `Wait for the environment to finish setting up and then start playing with Rafter.`

File: rafter/step1.md [hunk 2]
```diff
@@ -1,20 +1,7 @@
...
-Initialize Helm to get Tiller running in Minikube
-`kubectl -n kube-system create serviceaccount tiller`{{execute}}
-`kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller`{{execute}}
-`helm init --service-account=tiller`{{execute}}
-
-Check if Tiller is running
-`kubectl get deploy -n kube-system | grep tiller-deploy`{{execute}}
-
-Install Rafter from dedicated Helm repository
 `helm repo add rafter-charts https://rafter-charts.storage.googleapis.com`{{execute}}
```
Comment (derberg) [inline rafter/step1.md]: I suggest to make 2 steps, one `Add new Chart's repository to Helm` and the `Install Rafter`

File: rafter/intro.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## Meet Rafter
+
+Rafter is a solution for storing and managing different types of files. It uses MinIO as object storage and gives you the ability to manage files using Kubernetes custom resources. In brief, Rafter is a fully Kubernetes-native content managing system!
```
Comment (derberg) [inline rafter/intro.md]: no no no, Rafter is not a content management system :)
please remove `In brief, Rafter is a fully Kubernetes-native content managing system!`

File: rafter/step2.md [hunk 4]
```diff
@@ -1,47 +1,53 @@
...
-`export BUCKET_NAME=$(kubectl get bucket pages -o jsonpath='{.status.remoteName}')`{{execute}}
-
-Echo and then copy the link and open it in a browser to see the website works fine
-`echo https://[[HOST_SUBDOMAIN]]-31311-[[KATACODA_HOST]].environments.katacoda.com/$BUCKET_NAME/webpage/examples-master/asset-store/webpage/index.html`{{execute}}
\ No newline at end of file
+**Use Case 1 - Rafter as static sites host**
+
+In this scenario, you will create a Bucket and then push an Asset to it. The Asset is a package containing all static files needed to build a website, such as HTML, JS and CSS files. Follow these steps:
+
+1. Export a GitHub webpage URL of ready-to-use website sources.
```
Comment (derberg) [inline rafter/step2.md]:
```suggestion
1. Export variable with ready-to-use sources of a simple Website.
```

Please also apply this suggestion to other parts where you write about exporting variables, just write `Export variable`

File: rafter/step2.md [hunk 5]
```diff
@@ -1,47 +1,53 @@
...
+     bucketRef:
+       name: pages
+   EOF
+   ```{{execute}}
+
+4. Make sure that the Asset status is `Ready` which means that fetching, unpacking, and filtering is completed. Run:
+
+   `kubectl get assets webpage -o jsonpath='{.status.phase}'`{{execute}}
+
+5. Export the Bucket name as an environment variable:
```
Comment (derberg) [inline rafter/step2.md]:
```suggestion
5. Export variable with the name fo the Bucket available in the Status of the Bucket. Keep in mind that name of the Bucket in the storage is not exactly the same as the name of specific Bucker CR:
```

File: rafter/step3.md [hunk 6]
```diff
@@ -1,55 +1,62 @@
...
+
+4. Make sure that the Asset status is `Ready` which means that fetching and communication with the Front Matter Service is completed. Run:
+
+   `kubectl get assets markdown-file -o jsonpath='{.status.phase}'`{{execute}}
+
+5. Check the status of the Asset CR. Now it has an additional Metadata object with a set of values. In this scenario, there should be the `title` key with some value.
+
+   `kubectl get asset markdown-file -o jsonpath='{.status.assetRef.files[0].metadata.title}'`{{execute}}
+
+6. Make sure that the file is in storage and you can extract it. Export the Bucket name and the file name as environment variables:
```
Comment (derberg) [inline rafter/step3.md]: regarding bucket, change as in previous comments

**Review by derberg — APPROVED** [review]
just few remarks so we can quickly merge, make sure all works and then move to rafter, and on a pull request in rafter we can check if maybe more fixes are needed

Thanks a lot for the contribution 🚀

File: rafter/step2.md (hunk 4 above)
Comment (mmitoraj) [reply rafter/step2.md]: I will follow the "template" we usually use in Kyma:

```suggestion
1. Export the URL to ready-to-use sources of a simple website as an environment variable. Run:
```

**Review by mmitoraj — COMMENTED** [review]
Review suggestions applied.

**Pull request merged**